### PR TITLE
DHFPROD-4659: Visualize metadata of the document in card view.

### PR DESCRIPTION
**Pull request by timur-isangulov**
#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Added Tests
  

- ##### Reviewer:

- [x] Reviewed Tests

### CODE REVIEW

**Review by Sanjeevani19 — DISMISSED** [review]

File: marklogic-data-hub-central/ui/src/components/record-view/record-view.test.tsx [hunk 1]
```diff
@@ -1,13 +1,13 @@
 import React from 'react';
 import { render, fireEvent, waitForElement } from '@testing-library/react';
-import { entitySearch } from "../../../assets/mock-data/explore/entity-search";
+import { entitySearch } from "../../assets/mock-data/explore/entity-search";
 import { BrowserRouter as Router } from 'react-router-dom';
-import RawDataCardView from './raw-data-card-view';
+import RawDataCardView from './record-view';
```
Comment (rahulvudutala) [inline marklogic-data-hub-central/ui/src/components/record-view/record-view.test.tsx:5]: Can this be RecordCardView?

**Review by rahulvudutala — CHANGES_REQUESTED** [review]

File: marklogic-data-hub-central/ui/src/components/record-view/record-view.test.tsx (hunk 1 above)
Comment (timur-isangulov) [reply marklogic-data-hub-central/ui/src/components/record-view/record-view.test.tsx:5]: Renamed. Thanks.

Comment (rahulvudutala) [reply marklogic-data-hub-central/ui/src/components/record-view/record-view.test.tsx]: Its still RawDataCardView

File: marklogic-data-hub-central/ui/src/components/record-view/record-view.tsx [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+                </div>
+              </Card>
+            </div>
+          </Col>)) : <span></span>}
+      </Row>
+    </div>
+  );
+}
+
+export default RawDataCardView;
```
Comment (rahulvudutala) [inline marklogic-data-hub-central/ui/src/components/record-view/record-view.tsx:147]: Can we rename it to RecordCardView?

Comment (timur-isangulov) [reply marklogic-data-hub-central/ui/src/components/record-view/record-view.tsx]: Oh was missing this change due to rebasing. Fixed. Thanks.

**Review by rahulvudutala — DISMISSED** [review]

**Review by Sanjeevani19 — APPROVED** [review]

**Review by rahulvudutala — APPROVED** [review]

**Review by Sanjeevani19 — APPROVED** [review]

**Pull request merged**